### ISSUE TITLE
Use a less technicall word as replacement for *instance*

### ISSUE THREAD

**Issue by roipoussiere** [mastodon/mastodon#8349]
When I promote Mastodon or other federated services, I notice that the main difficulty for many non-tech people is to understand the word *instance*.

I'm not alone to think that *instance* is a technical term and should not be used for the general public, and needs a replacement. I suggest the word **_community_**.

It is semantically correct, while far easier to understand, to say *You need to choose your community*, *this user is on a different community*, etc.

The fact is, for many non-tech users, this specific word is a big step to join Mastodon. It's a kind of a magic word to put my interlocutor in stand-by mode. Many times I lost my audience just by pronouncing it, like:
- (...) firstly you need to choose your instance and then...
- ok, lol, geek stuff here, not for me.

Basically, Mastodon instances are communities and it's the most important thing to understand for a new user.

But feel free to suggest other alternatives in comments.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by trwnh** [comment]
To be technical while still keeping it simple, each instance is really its own *site* -- not always community-based, as some Mastodon instances are generalistic and not catering to any particular community. This would also help highlight the fact that really, all we're doing is making a bunch of web sites talk to each other. Most people understand what a "site" or "web site" is; this should be apparent from how widespread it is to call Twitter e.g. "the bird site", and not "the bird network" or "the bird community".

I do agree that the word "instance" is a little out-there for a lot of people, as most don't particularly care about the finer details of instantiation and server architecture.

**Comment by Laurelai** [comment]
I like the term community because thats the majority use case. Perhaps a field you can fill in somewhere but have community be a default?

**Comment by ashleyhull-versent** [comment]
there are a few of these..  "suspend" = nuke from orbit - suspend sounds like something you can unsuspend

**Comment by jeyemwey** [comment]
I like the term **Neighborhood** because of four reasons:

1. Everybody lives in some kind of neighborhood and knows this concept from real life.
2. The term projects a common divider over an instance (whether it's "only this club" or even just "we support everyone") as every neighborhood is special. Also, every neighborhood has its own major who manages the instance and sets rules and guidelines.
3. Neighborhoods are connected by streets (_"the fediverse"_) whereas communities often life seperated from each others (for example in ["Gated Communities"](https://events.ccc.de/2015/11/04/32c3-gated-communities/#32c3-gated-communities-english)).
4. Neighborhoods are superb for visualization in Videos or Slides, just draw a few houses and a marketplace, where as Communities and Instances are somewhat virtual.

While writing this, I realize that a **Village**  is a similar usable term, however a village seems to be more seperated from other villages, more rural and more isolated (which contradicts with 3.).

I am very interested in a broad conversation about this topic, please share your thoughts.

**Comment by trwnh** [comment]
the new term is "server". i still think "site" or "website" is better, but eh, it's no longer "instance" (at least officially)
